resty/package.json

```diff
@@ -12,6 +12,7 @@
     "react-dom": "^17.0.2",
     "react-if": "^4.0.1",
     "react-json-pretty": "^2.2.0",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "superagent": "^6.1.0",
     "web-vitals": "^1.1.1"
```

Comment:
- Would like to see a deployment link on your README, you can also delete all of the auto generated text off the README
- Would like to see that history added to see the history page and instructions on the help page